File: tfplus/tests/demo_test.py
# Copyright 2023 tfplus.

import unittest

from tfplus.python.demo import print_localtime


class DemoTest(unittest.TestCase):
    def test_demo(self):
        # TODO: fix print_localtime export issue.
        print_localtime()


if __name__ == "__main__":
    unittest.main()

File: tfplus/tfplus/cc/demo_test.cc
// Copyright 2023 tfplus.

#include "tfplus/cc/demo.h"
#include "gtest/gtest.h"


namespace {

TEST(DemoTest, DemoBasicTest) {
    tfplus::Demo dm;
    dm.print_localtime();
    EXPECT_EQ(true, true);
}

}  // namespace
